### DISCUSSION TITLE
On windows 10, will you open the screen for a while?

### DISCUSSION THREAD

**Discussion by zerx-lab** [zed-industries/zed#12038] [Feature Requests]
When I open zed on win10, the whole window will be white for about 0.5 seconds, and then zed will be displayed. How to solve it?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
